.github/workflows/app-react.yml

```diff
@@ -27,5 +27,21 @@ jobs:
         working-directory: ${{env.working-directory}}
       - run: npm run build --if-present
         working-directory: ${{env.working-directory}}
-      - run: npm test
+      - run: npm run coverage
         working-directory: ${{env.working-directory}}
+
+      - name: Codecov
+        uses: codecov/codecov-action@v1.0.7
+        with:
+          # User defined upload name. Visible in Codecov UI
+          name: PIMS
+          # Repository upload token - get it from codecov.io. Required only for private repositories
+          token: c02e6ccf-6e0f-47cd-842e-ab2f488933b8
```

Comment:
At some point I'll need to regenerate this token and find a way to provide it outside of source.

Reply:
yeah, this got already merged to source control so nothing we can do now but I believe GitHub provides a way to create secrets?

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

and

https://help.github.com/en/actions/configuring-and-managing-workflows/using-variables-and-secrets-in-a-workflow